pkgs/games/airshipper/default.nix

```
@@ -68,7 +63,7 @@ rustPlatform.buildRustPackage rec {
         libXcursor
       ];
     in
-    ''
+      ''
```

this doesn't seem right
```suggestion
    ''
```

---

You are right, indentation problems on my part